src/aap_eda/api/serializers/project.py

```diff
@@ -246,6 +249,23 @@ class Meta:
         fields = ["id", "extra_var"]
         read_only_fields = ["id"]
 
+    def to_representation(self, extra_var):
+        return {
+            "id": extra_var.id,
+            "extra_var": self.replace_vault_data(extra_var),
+        }
+
+    def replace_vault_data(self, extra_var):
+        data = {}
+        for key, value in yaml.safe_load(extra_var.extra_var).items():
+            data[key] = (
+                VAULTED_STRING
+                if isinstance(value, str) and ANSIBLE_VAULT_STRING in value
```

Comment:
it is safe to change the condition to `value.startwith(ANSIBLE_VAULT_STRING)`?

Reply:
No, the actual vaulted string doesn't start with `ANSIBLE_VAULT_STRING`